pkgs/development/libraries/libuv/default.nix

```diff
@@ -35,47 +37,48 @@ stdenv.mkDerivation (finalAttrs: {
 
   outputs = [ "out" "dev" ];
 
+  toDisable = [
```

Comment:
What do you think of renaming this variable `disabledTests`?

Reply:
I fully agree. Done